lnbits/wallets/alby.py

```diff
@@ -0,0 +1,133 @@
+import asyncio
+from typing import AsyncGenerator, Optional
+
+import httpx
+from loguru import logger
+
+from lnbits.settings import settings
+
+from .base import (
+    InvoiceResponse,
+    PaymentResponse,
+    PaymentStatus,
+    StatusResponse,
+    Unsupported,
+    Wallet,
+)
+
+
+class AlbyWallet(Wallet):
+    """https://guides.getalby.com/alby-wallet-api/reference/api-reference"""
+
+    def __init__(self):
+        endpoint = settings.alby_api_endpoint
+        wallet_key = settings.alby_admin_key
+
+        if not endpoint or not wallet_key or not settings.alby_api_key:
+            raise Exception("cannot initialize getalby")
+
+        self.wallet_key = wallet_key
```

Comment:
There is no `settings.alby_admin_key` now.

Let's just remove all lines with `wallet_key` as it is not used anywhere.

```suggestion

        if not endpoint or not settings.alby_api_key:
            raise Exception("cannot initialize Alby: please provide endpoint and api key")

```

Reply:
i just fixed this as you were posting the message!